cli.js

```diff
@@ -88,6 +88,7 @@ delete opts._args;
 
 const includeSourceMaps = opts.source_maps;
 const workingDir = opts.path || process.cwd();
+// @ts-ignore - Complains about opts.output maybe being undefined, but it defaults to string via parsed args.
```

Comment:
@morreman Any idea how to get rid of this? `opts.output` comes from parsing CLI arguments, and has a default value, but I can't get it to work properly...

Reply:
@emilbjorklund  By ensuring the `opts.output` the complaint will resolve. 
```suggestion
const outputFile = opts.output && pathModule.resolve(workingDir, opts.output);
```

Reply:
Cool, I'll try that!